lib/vanagon/optparse.rb

```diff
@@ -71,6 +71,11 @@ def initialize(banner, symbols = []) # rubocop:disable Metrics/AbcSize
                 "Only build listed COMPONENTs") do |components|
           @options[:only_build] = components
         end
+
+        opts.on("--build_with_docker",
```

Comment:
I wonder if this could be a `--build_with <tool>` option instead? Maybe a future thing.

Reply:
[nit] Would be nice to switch this to dashes instead of underbars, because I'm lazy

Reply:
I think switching this to `--build-with=docker` or `--build-with=make` (or whatever that ends up being called) sooner rather than later is important, so that command line options change as little as possible from release to release